frontend/uddhar-frontend/src/shared/components/Navbar.jsx

```
@@ -112,14 +158,16 @@ const Navbar = ({children}) => {
             className="self-end mb-4 mr-4 cursor-pointer"
             onClick={() => setShowMenu(false)}
           />
-          <nav className="flex flex-col gap-4">
+          <a className="flex flex-col gap-4">
             {navLinks.map((link, counter = 100) => (
               <p key={counter++} className="text-lg hover:text-gray-500">
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)